pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc

```diff
@@ -17,17 +16,51 @@ The configuration file enables you to import content from a Git repository into
 The {ProductShortName} configuration file defines the following parameters:
 
 * `server` The URL of the {ProductShortName} server.
-* `repositories` A multi-valued structure with the following child members:
+* `repository` The name of the repository. It is recommended not to include spaces in the repository name.
+* `variants` A multi-valued structure containing the `name, `path`, and `canonical` values.
+The `canonical` tags tell the search engine that of this group of pages with similar content, one of them is "official" and we want it to be considered the most relevant.
 +
-** `name` (optional) A unique ID that displays in URLs on the {ProductShortName} site when you view modules and assemblies.
+** `name` (optional) A unique ID that displays in URLs on the {ProductShortName} site when you view modules and assemblies. If no value is provided for `name`, this field is defined as *DEFAULT*.
 +
 For example, if you specify `name: rhel_docs`, and one of your files is named [filename]`foo.adoc`, the module's URL in {ProductShortName} is:
-`\http://<_{ProductShortName}__url>/content/repositories/**rhel_docs**/entities/modules/getting_started/foo.adoc.preview`
-** `modules` (optional) Patterns for modules that show up in the search results on {ProductShortName}.
+`\http://<_{ProductShortName}__url>/content/repositories/*repository*/entities/modules/getting_started/foo.adoc.preview`
+** `path` The path containing the resource files (for example attributes.adoc).
+** `canonical` Canonical is associated with variants. A module belongs to one or more variants, but there is only one canonical variant. Possible values: *true* or *false*.
```

Comment:
I don't want to torture you on this one but I'm still not sure when I should set the value to false. Can you think of an example use case?

Reply:
@emarcusRH 

**Variants:** Separate sets of output built on the same set of files.
Examples of variants:
- Document level: Installing RHV using the CLI; Installing RHV using Cockpit
- Module level: The same module is published in multiple Red Hat documents, e.g. Red Hat Decision Manager, Red Hat Process Automation Manager

Modules can have variants. 
Assemblies can have variants.
Documents can have variants (Well, this is just another kind of assembly.)

**canonical**
If a search engine sees multiple variants of the same piece of content (html page/group of pages) it wants to know what the "main" or "real" or "original" is to establish ranking, and to know that you're not trying to cheat somehow. Declaring a piece of content as `canonical` tells the search engine to treat that piece of content as the original, i.e. _canonical_ variant.

Declaring `canonical` to be true or false translates into HTML meta data. For example, let's say we have a module, snoopy, that appears in 3 different documents on example.com. The URLs for that module are as follows:

www.example.com/installing/snoopy_123 (canonical variant)
www.example.com/installing/snoopy_456
www.example.com/installing/snoopy_789
The canonical version

Here's the metadata that will appear in _all_ published variants. Notice that snoopy_123 is always declared as the canonical variant:

```
<link rel="canonical" href="www.example.com/installing/snoopy_123" />
<link rel="shortlink" href="www.example.com/installing/snoopy_456" />
<link rel="shortlink" href="www.example.com/installing/snoopy_789" />
```